### ISSUE TITLE
Black screen and default fullscreen mode

### ISSUE THREAD

**Issue by Boiethios** [Secretchronicles/TSC#542]
I built and installed the game from source:
```
tsc --version
This is The Secret Chronicles of Dr. M. version 2.0.0.
It was compiled from commit .
```

When I launch it, there is a black fullscreen, and nothing else. No error message. Just:
> Warning: Preferences file '/home/Boiethios/.config/tsc/config.xml' does not exist. Using default values.
I must kill the program from tty to unlock the situation.

I am under Debian testing / GNOME 3.20 with Xwayand session.

**Comment by Quintus** [comment]
the problem is known (ticket #500). It is an issue with CEGUI 0.7.9 and is being resolved during the CEGUI 0.8.x porting efforts, which you can follow in ticket #10 and the `feature-cegui-0.8` branch.

Vale,
Quintus

**Comment by Boiethios** [comment]
Sorry for the duplicate, I searched for a similar issue but did not found it.

Gratiam tibi ago.
